optaplanner-docs/src/main/asciidoc/PlannerIntroduction/DownloadAndRunTheExamples/DownloadAndRunTheExamples-section.adoc

```diff
@@ -40,48 +40,6 @@ OptaPlanner itself has no GUI dependencies.
 It runs just as well on a server or a mobile JVM as it does on the desktop.
 ====
 
-
```

Comment:
We always leave 2 empty lines before a section header (so "= foo" or "== foo" or ...) in adoc files.
That way it's easier to spot the section start and close